protocol/x/clob/keeper/orders.go

```diff
@@ -516,9 +554,31 @@ func (k Keeper) PerformOrderCancellationStatefulValidation(
 ) error {
 	orderIdToCancel := msgCancelOrder.GetOrderId()
 	if orderIdToCancel.IsStatefulOrder() {
-		cancelGoodTilBlockTime := msgCancelOrder.GetGoodTilBlockTime()
 		previousBlockTime := k.blockTimeKeeper.GetPreviousBlockInfo(ctx).Timestamp
 
+		prevBlockHeight := k.blockTimeKeeper.GetPreviousBlockInfo(ctx).Height
+		currBlockHeight := uint32(ctx.BlockHeight())
+		if lib.IsDeliverTxMode(ctx) && prevBlockHeight != currBlockHeight-1 {
+			k.Logger(ctx).Error(
+				fmt.Sprintf(
+					"PerformOrderCancellationStatefulValidation: prev block height is not one below current block height"+
+						"in DeliverTx. Prev: %d, Current: %d", prevBlockHeight, ctx.BlockHeight(),
```

Comment:
Nit: use `currBlockHeight` if possible
```suggestion
						"in DeliverTx. Prev: %d, Current: %d", prevBlockHeight, currBlockHeight,
```

Reply:
Should you add the `msgCancelOrder` in the error log?